content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -95,6 +96,9 @@ If you're crafting your own audit profile, you can use the audit profile for Goo
 [configure-helper.sh](https://github.com/kubernetes/kubernetes/blob/{{< param "githubbranch" >}}/cluster/gce/gci/configure-helper.sh)
 script, which generates an audit policy file. You can see most of the audit policy file by looking directly at the script.
 
+You can also refer to the [`Policy` reference](/docs/reference/config-api/apiserver-audit.v1/#audit-k8s-io-v1-Policy)
```

Comment:
nit: possibly:
You can also refer to the [`Policy` configuration reference](...) ...